scripts/download-rc.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
-HZ_VERSION="3.9"
+HZ_VERSION="3.10-SNAPSHOT"
```

Comment:
We should use the latest maintenance release instead of a snapshot unless we have to.